src/SearchResults.js

```diff
@@ -45,19 +45,21 @@ export default function Search({
   }, [searchValue, searchQuery]);
 
   return (
-    <SoundList
-      header={
-        showHeader && (
-          <Helmet>
-            <title>{`Search results for "${
-              searchQuery || searchValue
-            }"`}</title>
-            <meta name="description" content="A sound from Tide" />
-          </Helmet>
-        )
-      }
-      tracks={searchResults}
-      className="py-16"
-    />
+    <>
+      header=
+      {showHeader && (
+        <Helmet>
+          <title>{`Search results for "${searchQuery || searchValue}"`}</title>
+          <meta name="description" content="A sound from Tide" />
```

Comment:
I think we just need to change `content="A sound from Tide"` to `content="Search results from Tide"`. @amilajack might have more suggestions, but that's all I can think of currently.

Reply:
Oops sorry! I missed that! Will push that change now